config/crd/bases/ocs.openshift.io_managedocs.yaml

```
@@ -0,0 +1,65 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

We should be using 
```suggestion
apiVersion: apiextensions.k8s.io/v1
```
for OCP > 4.6

---

you are correct and this is actually very strange, as this was generated using operator-sdk v1.2 which presumably should generate  `apiextensions.k8s.io/v1` CRDs. Do you know how to instruct it to do so?

---

We can either upgrade controller-gen to 0.4.1 or pass a CRD version flag to 0.3.0 generator.

---

done